daemon/web/pages/containers/Containers.js

```diff
@@ -24,13 +26,19 @@ class ContainersWrapper extends React.Component {
     return (
       <div>
         <IconHeader type="containers" title="/inertia-deploy-test_dev_1" />
-        <div className="containerInfo" />
+        <div className="containerInfo" >
+          <Status title="Status:" status="Active" />
+          <h3>Last Updated:</h3>
+          <h4>{this.props.dateUpdated}</h4>
+        </div>
         <TerminalView logs={mocklogs} />
       </div>
     );
   }
 }
-ContainersWrapper.propTypes = {};
+ContainersWrapper.propTypes = {
```

Comment:
I think the linter might complain about the lack of an empty line here, but we'll have to wait for the builds to finish 🤷‍♂️